webclient/src/components/views/Create.vue

```diff
@@ -30,14 +33,16 @@ export default {
         CurrentIteration: {
           Tasks: []
         },
-        Completed: false
+        Completed: false,
+        loading: false
```

Comment:
I think you meant to make this part of the object returned, not the project object.

Reply:
Also correct.